CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -787,7 +787,8 @@ void IRenderWebBrowser.UpdateDragCursor(DragOperationsMask operation)
         /// <param name="operation">describes the allowed operation (none, move, copy, link). </param>
         protected virtual void UpdateDragCursor(DragOperationsMask operation)
         {
-            //TODO: Someone should implement this
+            var dragCursor = DragCursorProvider.GetCursor(operation);
+            UiThreadRunSync(() => Mouse.SetCursor(dragCursor));
```

Comment:
Is there a reason this is run `Sync`? If so please add a comment as to the reason.

Reply:
This is an attempt to reduce the flicker, but it still exists.